pkg/controllers/mcs/service_export_controller.go

```diff
@@ -190,6 +190,12 @@ func (c *ServiceExportController) enqueueReportedEpsServiceExport() {
 			continue
 		}
 
+		err = c.waitEpsSyncedOnCluster(context.TODO(), clusterName, 1*time.Second)
+		if err != nil {
+			// never reach here
+			continue
+		}
```

Comment:
It looks weird to wait for the cache synced here.

cc @zach593 
Can you help to take a look?

Reply:
In the current implementation of informerManager, checking whether the cache has synced is unavoidable. Personally, I feel it would be better to perform the check or wait before using the informer, as this would enhance the cohesion of the informerManager logic. Also, it seems this approach wouldn’t increase the number of checks or waits, since the current PR also performs a check for each Work resource.

In the long term, we could consider refactoring informerManager to behave more like controller-runtime, blocking until the cache is fully synced, or times out, or the informerManager for that cluster is closed.